babycare_product_golive_checks/models/product.py

```diff
@@ -0,0 +1,264 @@
+# -*- coding: utf-8 -*-
+import logging
+from openerp import models, api
+from openerp.tools.translate import _
+
+_logger = logging.getLogger(__name__)
+
+
+class ProductTemplate(models.Model):
+    _inherit = 'product.template'
+
+    @api.model
+    def check_ready_for_magento(self):
+        text = ''
+        not_image_ids = []
+        not_magento_type_ids = []
+        not_short_description_ids = []
+        not_long_description_ids = []
+        not_ean13_ids = []
+        not_reference_ids = []
+        not_weight_ids = []
+        not_pubcategorie_ids = []
+        not_published_ids = []
+        not_magento_attset_ids = []
+        not_wk_website_ids = []
+        not_variant_reference_ids = []
+        not_variant_ean13_ids = []
+        success_ids = []
+        for template_id in self._context.get('active_ids'):
+            obj = self.env['product.template'].browse(template_id)
+            template_name = obj.name
+            image = obj.image_medium
+            odoo_type = obj.type
+            magento_type = obj.prod_type
+            short_description = obj.description_sale
+            long_description = obj.description
+            ean13 = obj.ean13
+            reference = obj.default_code
+            weight = obj.weight_net
+            pubcategorie = obj.public_categ_ids
+            website_published = obj.website_published
+            magento_attset = obj.attribute_set_id
+            wk_websites_id = obj.wk_websites_id
+            if not image:
```

Comment:
This line and the lines below are more readable if you call the browse record `template` instead of `obj` and refer to template.image_medium directly here, skipping the mass assignment above.